src/components/Restoration/RestorationType.vue

```diff
@@ -102,6 +111,17 @@ export default class RestorationType extends Vue {
     return (this.getShowErrors && !this.getRestorationTypeValid)
   }
 
+  /**
+   * Default State (if applicable) when we create new Restoration.
+   * No radio button is selected.
+   */
+  mounted (): void {
```

Comment:
When we're creating a new restoration, we create a new full restoration with no relationships. The UX team wants the default state to no radio button being selected. This deals with that.

Reply:
Fee summary in default state:
![fee summary in default state](https://user-images.githubusercontent.com/122301442/222843185-03d9d794-6c6b-4aaa-9260-65d52d0b2c4c.PNG)

Reply:
I don't think clearing the restoration type (as below) is the right way to handle this. We should instead NOT set the initial restoration draft to Full. That will probably break other things, like the draft filing in the Todo List, but that's an easy fix. Can you identify anything else that will break?

Reply:
Thing is, I didn't want to go into that rabbit hole of changing Filings UI for this specific case of default state. I'm trying now to fix it in Filings UI but to no avail. I will look into it more.

Reply:
There are a couple of code smells here. First, the special-case logic (which additionally only happens at a particular point in time -- when the component is mounted). Second, the double fee codes (I think it should show no fee codes if a type hasn't been selected).

If the UX team wants the default to be "neither full nor limited resto" then let's handle that directly. But are we saying that if the user doesn't select any relationships, saves the draft, and resumes it, that we should clear their previous full resto selection? That doesn't sound right to me.

I think the thing to do here is separately handle the full/limited selection from the relationships selection.

In Filings UI, the Type is defaulted when the initial draft is created. You should be able to just remove that, but as I said, a few areas will need to be updated to handle "no" Type selected. Shouldn't be too bad though. LMK if I can help.

One last thing: if this is a significant updated requirement then we may need to review the impacts and push back a bit, maybe create a new ticket and prioritize it against all the other tickets. This feels a bit out of scope for "just some UX tweaks".

Reply:
Agreed. I was thinking, shall merge this PR without the default state change and create another ticket that will handle that? This is because this involves changing Filings UI and moreover, it's not an "UX" change. 

Also, having not worked with Filings UI before, I just a little bit of time to get familiar with the codebase there. Shouldn't take too long though.

Reply:
I'm in favour of leaving the initial restoration type out of this PR.

PS - in Filings UI, start here: https://github.com/bcgov/business-filings-ui/blob/0b49f6659e120bef4acc9f9bfc09f1f8855b8cc3/src/components/Dashboard/StaffNotation.vue#L121

Reply:
Alright, sounds good. Thanks Sev 😄